examples/typescript/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "typescript-example",
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "test": "tsc --noEmit"
```

Comment:
Need to add this command to travis tasks. Will need help here.

Reply:
this is the important bit: https://github.com/Nozbe/WatermelonDB/blob/master/.travis.yml#L31 -- so if you add to the root package.json something like `test:typescript = ... cd examples/... yarn testst` and then to `ci:check` add `previous commands && yarn test:typescript` that śhould work

Reply:
Done.